testing/integration/fqdn_test.go

```diff
@@ -103,8 +103,14 @@ func (s *FQDN) TestFQDN() {
 			kibana.MonitoringEnabledLogs,
 			kibana.MonitoringEnabledMetrics,
 		},
+		AgentFeatures: []map[string]interface{}{
+			{
+				"name":    "fqdn",
+				"enabled": false,
+			},
+		},
```

Comment:
Why this change? What does it fix? Is `enabled: false` not the default?

Reply:
So, at least on earlier iterations of the Kibana helper libraries, the API call would fail if `AgentFeatures` was empty.